terraform/lang/security/iam/no-iam-data-exfiltration.yaml

```diff
@@ -61,8 +74,7 @@ rules:
     - https://cloudsplaining.readthedocs.io/en/latest/glossary/data-exfiltration/
     category: security
     license: Commons Clause License Condition v1.0[LGPL-2.1-only]
-  paths:
-    include:
-    - '*.tf'
-  languages: [generic]
+    technology:
```

Comment:
is it possible to add CWE number here?
(I would propose smth like https://cwe.mitre.org/data/definitions/200.html, but you know better)